app/pages/help.html

```diff
@@ -678,6 +678,8 @@ <h3 id="firefoxmode">firefoxmode</h3>
         With the Firefox mode setting you can make Vieb present itself to the websites as if you are using Firefox. It will update the useragent and other navigator properties. You can either enable this for all websites, or only for Google websites to circumvent Google's useragent checks at login.
         <h3 id="fontsize">fontsize</h3>
         Change the font size for the entire Vieb interface. Changing the font size setting will also scale the navbar, built-in pages such as this help page, notifications and <a class="follow" href="#action.startFollowCurrentTab">follow mode</a> indicators. It will not automatically zoom websites, which can be done with <a href="#action.zoomIn">action.zoomIn</a> and related actions.
+        <h3 id="followelement">followelement</h3>
+        This setting can be used to filter the types of elements that will be highlighted when using the <a href="#action.startFollowCurrentTab">follow mode</a>. Different sets of elements can be selected at any moment to be highlighted. For example, with this setting set on "inputs-click,inputs-insert", only input elements will be highlighted in follow mode. Please check <a href="#action.startFollowCurrentTab">follow mode</a> for more information about the types of elements.
```

Comment:
Can you give the `a` tags a `class="follow"` to highlight them in the follow mode color? Thanks!